src/compiler/diagnosticMessages.json

```
@@ -7356,5 +7356,9 @@
     "A 'return' statement cannot be used inside a class static block.": {
         "category": "Error",
         "code": 18041
+    },
+    "An interface cannot extend a primitive type like '{0}'; An interface can only extend named types and classes.": {

```

Don't capitalize after the semicolon.

```suggestion
    "An interface cannot extend a primitive type like '{0}'. An interface can only extend named object types.": {
```

---

Changes:
* Error messages were rewritten to standard